envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/EnvoyEgressRoutesFactory.kt

```diff
@@ -88,7 +97,10 @@ internal class EnvoyEgressRoutesFactory(
                     )
                 }
             }
-            .build()
+        if (addUpstreamAddressHeader)
```

Comment:
Little style nitpick, in a line 116 we use curly braces in single line if statement. I thought we could configure detekt/ktlint rule in order to maintain consistent style.

Reply:
Done. I had to do some additional changes after changing detekt config - there was no curly braces in few if statements in our existing code.